library/Fission/Web/Heroku/Deprovision.hs

```diff
@@ -17,6 +17,11 @@ import           Servant
 import           Fission.Prelude
 import           Fission.Models
 
+import qualified Fission.User.Query        as UserQuery
+import qualified Fission.User.Mutation     as UserMutation
```

Comment:
We can also re-export these from `Fission.User`, &c. Then it would read like:


```haskell
userCids <- UserCID.getByUserId (entityKey addOnUser)
-- and
User.destroy userId
```

Reply:
Hmm i like that